docs/spec/spec.md

```diff
@@ -50,6 +50,9 @@ metadata:
   namespace: default
   labels:
     knative.dev/service: ...  # name of the Service automatically filled in
+  annotations:
+    serving.knative.dev/creator: ...       # the user identity who created the service.
```

Comment:
Can you note that these are system-generated?

Reply:
Done.